src/main/java/net/dv8tion/jda/internal/JDAImpl.java

```diff
@@ -82,10 +83,12 @@
 import net.dv8tion.jda.internal.utils.config.ThreadingConfig;
 import okhttp3.OkHttpClient;
 import okhttp3.RequestBody;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Wrong annotation used? Given there is already the Nonnull from javax.

Reply:
I fixed that already just missed the import. Thanks for spotting it.